subprojects/docs/src/snippets/dependencyManagement/catalogs-javaPlatformUsage/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// tag::platform_uses_catalog[]
+plugins {
+    `java-platform`
+}
+
+dependencies {
+    constraints {
+        api(libs.mylib) { version { require("1.1") } }
```

Comment:
This `{ version { require("1.1") } }` does not look like it belongs here

Reply:
good catch, it's no harm but it's inconsistent with the Groovy example